### PR TITLE
updated extern files

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -39,7 +39,6 @@ install_requires =
   pandas>=0.23.0
   astropy>=4.0
   parfive[ftp]>=1.1.0
-  importlib_metadata;python_version<"3.8"
```
Comment (nabobalis) [inline setup.cfg:42]: Not sure we are using this for anything anymore?

Comment (kakirastern) [reply setup.cfg:42]: The package `importlib_metadata` is intended to be ported to Python 3.8... But then something is off (or fishy) if that is the case.

Comment (Cadair) [reply setup.cfg:42]: I can't remember what we were using this for in the first place?

Comment (nabobalis) [reply setup.cfg:42]: We were using to to check the python version according to the changelog.

Comment (kakirastern) [reply setup.cfg:42]: I see. Thanks @nabobalis!

**Review by kakirastern — APPROVED** [review]

**Review by Cadair — APPROVED** [review]

**Pull request merged**